cmd/istio_ca/main.go

```diff
@@ -172,14 +174,17 @@ func createCA() ca.CertificateAuthority {
 	caOpts := &ca.IstioCAOptions{
 		CertChainBytes:   certChainBytes,
 		CertTTL:          opts.certTTL,
+		Core:             core,
+		Namespace:        opts.namespace,
```

Comment:
What happens when namespace is unspecified (i.e., in cluster-wide deployment)?

Reply:
Fixed.